feature/problem.feature

```diff
@@ -0,0 +1,78 @@
+Feature: Problem solving
+    A person can solve a problem by asking experts.
+
+    Scenario: a new problem creation
+        Given there is a person named "Alex"
+        When "Alex" creates a problem "How to build a decision making system"
+        And there is no similar problems yet
+        Then the problem is created
+        And the new "Route" is added to the problem
+        And the new "Round Table" is added to the route
+
+    Scenario Outline: ability to reuse a route of an already existing problem
+        Given there is a problem "How to build a decision making system"
+        And this problem has a "Route"
+        And this "Route" has a "Round Table"
+        When "Dima" creates a similar problem "How to build an automated decision making system"
+        Then he receives a suggestion to reuse the "Route" for the existing problem
+        And he receives a suggestion to reuse the "Round Table" for the existing route
+        And he makes a <decision>
+        And he gets the <result>
+        Examples:
+            | decision | result      |
+            | accept   | reuse_route |
+            | decline  | new_problem |
+
+    Scenario Outline: ability to reuse a round table of an already existing problem
+        Given there is a problem "How to build a decision making system"
+        And this problem has a "Route"
+        And this "Route" has a "Round Table"
+        When "Dima" creates a similar problem "How to build an automated decision making system"
+        Then he receives a suggestion to reuse the "Route" for the existing problem
+        When he accepts this suggestion
+        Then he receives a suggestion to reuse the "Round Table" for the accepted route
+        And he makes a <decision>
+        And he gets the <result>
+        Examples:
+            | decision | result            |
+            | accept   | reuse_round_table |
+            | decline  | new_round_table   |
+
+    Scenario: grouping similar round tables to a virtual route
```

Comment:
### (It's correct, but it can also happen between routes of different "problems.") E.g.   
problem 1 : there are few parks in the town
A Route is created, several Round Tables are created, people discuss about various topics, in some of the Round Tables emerges the problem that the town doesn't receive enough money from central administration.  

problem2: there are not enough studying room open during the week-end.   
A Route is created, several Round Tables are created, people discuss about various topics, in some of the Round Tables emerges the problem that there are not enough money to pay for the people that work in the study rooms, so that is why they are closed during week-end.   
  
Different original problems, different Routes, different Round Tables, but the issue is related, so that's why a Virtual Route is created, and comprehend the Round Tables that have that topic in common.

Reply:
I simplified the scenario so that it becomes evenly said that virtual grouping can happen for either cases: the same problem related round tables, or different problem related round tables.